ui/src/routes/_layout/create-organization.tsx

```diff
@@ -58,7 +58,10 @@ const CreateOrganizationPage = () => {
     onSuccess() {
       toast({
         title: 'Create Organization',
-        description: 'New organization created successfully.',
+        description:
+          'New organization "' +
+          form.getValues().name +
+          '" created successfully.',
```

Comment:
Just wondering, can / should [template literals](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Template_literals) be used here to improve readability via string interpolation? Like

    `New organization "${form.getValues().name}" created successfully.`

Reply:
I've never been a fan of those, as I'm not seeing why they would be more readable.

Reply:
We should have a clear line here, and do it (or not) consistently, if at all. I'm curious about @mmurto's advice about best practices here.

Reply:
As I'm sure he's pro-literals, I've already made the changes.